.github/workflows/e2e-build-images.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
               force="--force"
           fi
 
-          IMAGE_VERSION=$(go run ./e2e/cmd/00_check_vm_image --codename ${{ matrix.codename }} ${force})
+          IMAGE_VERSION=$(go run ./e2e/cmd/build_image/00_check_vm_image --codename ${{ matrix.codename }} ${force})
```

Comment:
On the directories arrangement, should not that be actually 3 directories? I can see:
- `build_base_image`
- `{provision;prepare}_vm` (or any other better name), where I would put the adsys deb build + provision client
- `run_tests` which will maybe not have any subdirectories and all the e2e tests here?

Would that makes sense?